src/components/index.js

```diff
@@ -0,0 +1,3 @@
+import Header from "./Header/Header";
+
+export { Header };
```

Comment:
you can directly export it like 
`export { Header } from "./Header/Header"` (if its a named export)